flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/IntermediateResultPartitionID.java

```diff
@@ -30,6 +30,7 @@
 public class IntermediateResultPartitionID implements ResultID {
 
 	private static final long serialVersionUID = 1L;
+	private static final int BYTEBUF_LEN = 20;
```

Comment:
similarly here, please add a comment. 
Where are the extra 4 bytes coming from? partitionNum?

Reply:
Yes, the extra 4 bytes come from the `partitionNum`.